### PR TITLE
Dodaje małpi hełm, skiny borgów + jakąś tam krew

### PR DESCRIPTION
**Pull request by MACIEKBAKI**
## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

## Changelog
:cl:
add: skiny borguf
add: Zasyganie krfi
add: małpi hełm
/:cl:

### CODE REVIEW

**Comment by MACIEKBAKI** [comment]
nie mergeować

**Comment by MACIEKBAKI** [comment]
merdż redi

File: aquila/code/modules/mob/living/carbon/monkey/monkey.dm [hunk 1]
```diff
@@ -1,11 +1,10 @@
 /mob/living/carbon/monkey
 	speech_sound = "monkey"
 
-/mob/living/carbon/monkey/IsAdvancedToolUser()//Unless its monkey mode monkeys cant use advanced tools
-	if(mind && is_monkey(mind))
-		return TRUE
-	return FALSE
-
+/mob/living/carbon/monkey/IsAdvancedToolUser()
```
Comment (MACIEKBAKI) [inline aquila/code/modules/mob/living/carbon/monkey/monkey.dm:4]:
```suggestion
/mob/living/carbon/monkey/IsAdvancedToolUser() // AQ EDIT
```

File: code/__DEFINES/misc.dm [hunk 2]
```diff
@@ -106,11 +106,18 @@ GLOBAL_LIST_EMPTY(bloody_footprints_cache)
...
 #define BLOOD_LOSS_IN_SPREAD		20
 #define BLOOD_AMOUNT_PER_DECAL		20
 
+// AQ EDIT
+#define BLOOD_DRY_TIME_MINIMUM        60 SECONDS
+#define BLOOD_DRY_TIME_PER_BLOODINESS 20 SECONDS
+#define BLOOD_SLIPPERY_KNOCKDOWN      3 SECONDS
+#define BLOOD_SLIPPERY_TRESHOLD       (MAX_SHOE_BLOODINESS / 2)
+#define BLOOD_DRY_COLOR               rgb(160, 160, 160, 255)
+
```
Comment (MACIEKBAKI) [inline code/__DEFINES/misc.dm:120]:
```suggestion
#define BLOOD_GAIN_PER_STEP			40 // AQ EDIT (100
#define BLOOD_LOSS_PER_STEP			4 // AQ EDIT (5)
#define BLOOD_LOSS_IN_SPREAD		20
#define BLOOD_AMOUNT_PER_DECAL		20

// AQ EDIT
#define BLOOD_DRY_TIME_MINIMUM        60 SECONDS
#define BLOOD_DRY_TIME_PER_BLOODINESS 20 SECONDS
#define BLOOD_SLIPPERY_KNOCKDOWN      3 SECONDS
#define BLOOD_SLIPPERY_TRESHOLD       (MAX_SHOE_BLOODINESS / 2)
#define BLOOD_DRY_COLOR               rgb(160, 160, 160, 255)
// AQ EDIT END
```

File: code/_globalvars/traits.dm [hunk 3]
```diff
@@ -15,7 +15,7 @@ GLOBAL_LIST_INIT(traits_by_type, list(
 		"TRAIT_BADDNA" = TRAIT_BADDNA,
 		"TRAIT_CLUMSY" = TRAIT_CLUMSY,
 		"TRAIT_DUMB" = TRAIT_DUMB,
-		"TRAIT_MONKEYLIKE" = TRAIT_MONKEYLIKE,
+		"TRAIT_DISCOORDINATED" = TRAIT_DISCOORDINATED,
```
Comment (MACIEKBAKI) [inline code/_globalvars/traits.dm:18]:
```suggestion
		"TRAIT_DISCOORDINATED" = TRAIT_DISCOORDINATED, // AQ EDIT (TRAIT_MONKEYLIKE -> TRAIT_DISCOORDINATED)
```

File: code/datums/brain_damage/severe.dm [hunk 4]
```diff
@@ -227,11 +227,11 @@
 	lose_text = "<span class='notice'>You feel in control of your hands again.</span>"
 
 /datum/brain_trauma/severe/discoordination/on_gain()
-	ADD_TRAIT(owner, TRAIT_MONKEYLIKE, TRAUMA_TRAIT)
+	ADD_TRAIT(owner, TRAIT_DISCOORDINATED, TRAUMA_TRAIT)
 	..()
 
 /datum/brain_trauma/severe/discoordination/on_lose()
-	REMOVE_TRAIT(owner, TRAIT_MONKEYLIKE, TRAUMA_TRAIT)
+	REMOVE_TRAIT(owner, TRAIT_DISCOORDINATED, TRAUMA_TRAIT)
```
Comment (MACIEKBAKI) [inline code/datums/brain_damage/severe.dm:234]:
```suggestion
	REMOVE_TRAIT(owner, TRAIT_DISCOORDINATED, TRAUMA_TRAIT) // AQ EDIT (TRAIT_MONKEYLIKE -> TRAIT_DISCOORDINATED)
```

File: code/game/objects/effects/decals/cleanable.dm [hunk 5]
```diff
@@ -85,13 +85,10 @@
...
-			if(bloodiness >= BLOOD_GAIN_PER_STEP)
-				add_blood = BLOOD_GAIN_PER_STEP
-			else
-				add_blood = bloodiness
-			bloodiness -= add_blood
-			S.bloody_shoes[blood_state] = min(MAX_SHOE_BLOODINESS,S.bloody_shoes[blood_state]+add_blood)
+			// AQ EDIT
+			//always non-positive
+			var/change = add_bloodiness(-BLOOD_GAIN_PER_STEP)
+			S.bloody_shoes[blood_state] = min(MAX_SHOE_BLOODINESS, S.bloody_shoes[blood_state] - change)
```
Comment (MACIEKBAKI) [inline code/game/objects/effects/decals/cleanable.dm:91]:
```suggestion
			// AQ EDIT
			//always non-positive
			var/change = add_bloodiness(-BLOOD_GAIN_PER_STEP)
			S.bloody_shoes[blood_state] = min(MAX_SHOE_BLOODINESS, S.bloody_shoes[blood_state] - change)
			// AQ EDIT END
```

File: code/game/objects/effects/decals/cleanable/humans.dm [hunk 6]
```diff
@@ -41,20 +36,23 @@
...
 	icon_state = "tracks"
 	random_icon_states = null
 
-/obj/effect/decal/cleanable/trail_holder //not a child of blood on purpose
-	name = "blood"
+/obj/effect/decal/cleanable/blood/trail_holder
+	name = "bloody trails"
 	icon = 'icons/effects/blood.dmi'
+	icon_state = null //rendered through overlays
+	random_icon_states = null
```
Comment (MACIEKBAKI) [inline code/game/objects/effects/decals/cleanable/humans.dm:51]:
```suggestion
/obj/effect/decal/cleanable/blood/trail_holder // AQ EDIT - made a child of blood
	name = "bloody trails"
	icon = 'icons/effects/blood.dmi'
	icon_state = null //rendered through overlays
	random_icon_states = null
```

File: code/game/objects/effects/decals/cleanable/humans.dm [hunk 7]
```diff
@@ -66,24 +64,18 @@
 	random_icon_states = list("gib1", "gib2", "gib3", "gib4", "gib5", "gib6")
 	mergeable_decal = FALSE
 	turf_loc_check = FALSE
+	is_slippery = TRUE
```
Comment (MACIEKBAKI) [inline code/game/objects/effects/decals/cleanable/humans.dm:67]:
```suggestion
	is_slippery = TRUE // AQ EDIT
```

File: code/game/objects/items/storage/storage.dm [hunk 8]
```diff
@@ -62,14 +62,19 @@
 	return dat
 
 /obj/item/storage/compile_monkey_icon()
```
Comment (MACIEKBAKI) [inline code/game/objects/items/storage/storage.dm:64]:
```suggestion
/obj/item/storage/compile_monkey_icon() // AQ EDIT - made it support multiple icon_states per item
```

File: code/modules/clothing/head/_head.dm [hunk 9]
```diff
@@ -75,14 +75,17 @@
 		M.update_inv_head()
 
 /obj/item/clothing/head/compile_monkey_icon()
```
Comment (MACIEKBAKI) [inline code/modules/clothing/head/_head.dm:77]:
```suggestion
/obj/item/clothing/head/compile_monkey_icon() // AQ EDIT - made it support multiple icon_states per item
```

File: code/modules/clothing/under/_under.dm [hunk 10]
```diff
@@ -229,14 +229,19 @@
 	dying_key = DYE_REGISTRY_UNDER
 
 /obj/item/clothing/under/compile_monkey_icon()
```
Comment (MACIEKBAKI) [inline code/modules/clothing/under/_under.dm:231]:
```suggestion
/obj/item/clothing/under/compile_monkey_icon() // AQ EDIT - made it support multiple suggestions per item
```

File: code/modules/mob/living/blood.dm [hunk 11]
```diff
@@ -276,8 +276,7 @@
 		B = new /obj/effect/decal/cleanable/blood/splatter(T, get_static_viruses())
 	if(QDELETED(B)) //Give it up
 		return
-	if (B.bloodiness < MAX_SHOE_BLOODINESS) //add more blood, up to a limit
-		B.bloodiness += BLOOD_AMOUNT_PER_DECAL
+	B.add_bloodiness(BLOOD_AMOUNT_PER_DECAL)
```
Comment (MACIEKBAKI) [inline code/modules/mob/living/blood.dm:279]:
```suggestion
	B.add_bloodiness(BLOOD_AMOUNT_PER_DECAL) // AQ EDIT
```

File: code/modules/mob/living/silicon/robot/robot_modules.dm [hunk 12]
```diff
@@ -274,6 +274,9 @@
 	R.SetLockdown(TRUE)
 	R.anchored = TRUE
 	R.logevent("Chassis configuration has been set to [name].")
+	var/datum/effect_system/smoke_spread/smoke = new
+	smoke.set_up(1, R.loc)
+	smoke.start()
 	sleep(1)
```
Comment (MACIEKBAKI) [inline code/modules/mob/living/silicon/robot/robot_modules.dm:280]:
```suggestion
	R.logevent("Chassis configuration has been set to [name].")
	// AQ EDIT START
	var/datum/effect_system/smoke_spread/smoke = new
	smoke.set_up(1, R.loc)
	smoke.start()
	// AQ EDIT END
	sleep(1)
```

File: code/modules/mob/living/simple_animal/hostile/megafauna/bubblegum.dm [hunk 13]
```diff
@@ -355,6 +355,8 @@ Difficulty: Hard
 
 /obj/effect/decal/cleanable/blood/bubblegum
 	bloodiness = 0
+	can_dry = FALSE
+	is_slippery = FALSE
```
Comment (MACIEKBAKI) [inline code/modules/mob/living/simple_animal/hostile/megafauna/bubblegum.dm:359]:
```suggestion
	can_dry = FALSE // AQ EDIT
	is_slippery = FALSE // AQ EDIT
```

File: code/modules/mob/living/simple_animal/hostile/megafauna/bubblegum.dm [hunk 14]
```diff
@@ -406,6 +408,8 @@ Difficulty: Hard
 	desc = "Thick, splattered blood."
 	random_icon_states = list("gib3", "gib5", "gib6")
 	bloodiness = 20
+	can_dry = FALSE
+	is_slippery = FALSE
```
Comment (MACIEKBAKI) [inline code/modules/mob/living/simple_animal/hostile/megafauna/bubblegum.dm:412]:
```suggestion
	can_dry = FALSE // AQ EDIT
	is_slippery = FALSE // AQ EDIT
```

File: code/modules/tgui/status_composers.dm [hunk 15]
```diff
@@ -50,7 +50,7 @@
 /// Returns a UI status such that advanced tool users will be able to update,
 /// but everyone else can only watch.
 /proc/ui_status_user_is_advanced_tool_user(mob/user, atom/source)
-	return (user.canUseTopic(source, BE_CLOSE) && !HAS_TRAIT(user, TRAIT_MONKEYLIKE)) ? UI_INTERACTIVE : UI_UPDATE
+	return (user.canUseTopic(source, BE_CLOSE) && !HAS_TRAIT(user, TRAIT_DISCOORDINATED)) ? UI_INTERACTIVE : UI_UPDATE
```
Comment (MACIEKBAKI) [inline code/modules/tgui/status_composers.dm:53]:
```suggestion
	return (user.canUseTopic(source, BE_CLOSE) && !HAS_TRAIT(user, TRAIT_DISCOORDINATED)) ? UI_INTERACTIVE : UI_UPDATE // AQ EDIT (TRAIT_MONKEYLIKE -> TRAIT DISCOORDINATED)
```

File: config/config.txt [hunk 16]
```diff
@@ -71,7 +71,7 @@ MENTOR_LEGACY_SYSTEM
 #MENTORS_MOBNAME_ONLY
 
 ## Comment this out to stop locally connected clients from being given the almost full access !localhost! admin rank
-#ENABLE_LOCALHOST_RANK
+ENABLE_LOCALHOST_RANK
```
Comment (MACIEKBAKI) [inline config/config.txt:74]:
```suggestion
#ENABLE_LOCALHOST_RANK
```

File: nsv13/code/game/objects/items/bloody_eye.dm [hunk 17]
```diff
@@ -158,7 +158,7 @@
 
 /datum/status_effect/bloody_eye/on_apply()
 	to_chat(owner, "<span class='warning'>KILLKILLKILL</span>")
-	ADD_TRAIT(owner,TRAIT_MONKEYLIKE,type)
+	ADD_TRAIT(owner,TRAIT_DISCOORDINATED,type)
```
Comment (MACIEKBAKI) [inline nsv13/code/game/objects/items/bloody_eye.dm:161]:
```suggestion
	ADD_TRAIT(owner,TRAIT_DISCOORDINATED,type) // AQ EDIT (TRAIT_MONKEYLIKE -> TRAIT DISCOORDINATED)
```

File: nsv13/code/game/objects/items/bloody_eye.dm [hunk 18]
```diff
@@ -176,7 +176,7 @@
 	owner.clear_fullscreen("brute")
 	owner.adjustOxyLoss(25,0)
 	owner.remove_movespeed_modifier(type)
-	REMOVE_TRAIT(owner,TRAIT_MONKEYLIKE,type)
+	REMOVE_TRAIT(owner,TRAIT_DISCOORDINATED,type)
```
Comment (MACIEKBAKI) [inline nsv13/code/game/objects/items/bloody_eye.dm:179]:
```suggestion
	REMOVE_TRAIT(owner,TRAIT_DISCOORDINATED,type) // AQ EDIT (TRAIT_MONKEYLIKE -> TRAIT DISCOORDINATED)
```

**Review by Joker66613 — APPROVED** [review]

**Pull request merged**